packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -445,18 +445,6 @@ ensureIndex(Posts,
     name: "posts.afSticky_score",
   }
 );
-ensureIndex(Posts,
```

Comment:
we no longer really use the "meta" flag on posts, and I think it should be formally deprecated.